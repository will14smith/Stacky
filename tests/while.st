main () -> () { 5 { dup 0 > } { dup print 1 - } while drop }