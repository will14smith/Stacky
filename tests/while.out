=== Output ===
5
4
3
2
1

=== Exit ===
0
